tests/integration/components/polaris-resource-list/bulk-actions-test.js

```diff
@@ -0,0 +1,12 @@
+// import { module, test } from 'qunit';
+// import { setupRenderingTest } from 'ember-qunit';
+// import { render } from '@ember/test-helpers';
+// import hbs from 'htmlbars-inline-precompile';
+
+// module('Integration | Component | polaris-resource-list/bulk-actions', function(hooks) {
+//   setupRenderingTest(hooks);
+
+//   test('it renders', async function(assert) {
+
+//   });
+// });
```

Comment:
Port the tests across from the React implementation pls 😅

Reply:
<img width="75" alt="screen shot 2018-12-03 at 4 53 24 pm" src="https://user-images.githubusercontent.com/2292367/49403964-1cda1780-f71c-11e8-8586-c267a8b7b7ce.png">